egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -229,12 +278,20 @@ def ParseSpliceString(splice_indexes):
             }
 
```

Comment:
It might be better to add a comment on the top of the MakeConfigs saying we constantly change the function signature of MakeConfigs as it is intended just for local use in this script. We don't intend this function to be imported into other modules.